CONTRIBUTING.md

```diff
@@ -40,6 +40,11 @@ contains some questionable code itself! Also before making a pull request, pleas
 `util/update_lints.py`, which will update `lib.rs` and `README.md` with the lint declarations. Our
 travis build actually checks for this.
 
+Clippy uses UI tests. UI tests check that the output of the compiler is exactly as expected.
+Of course there's little sense in writing the output yourself or copying it around.
+Therefore you can simply run `tests/ui/update-all-references.sh` and check whether
+the output looks as you expect with `git diff`. Commit all `.stderr` files, too.
```

Comment:
:+1: Nice of you to update that!

I'd just say “Commit all `*.stderr` files” because I though for a moment “why the hell those are hidden files?”.